### PR TITLE
Ignore H102 when linting notebooks

### PR DESCRIPTION
**Pull request by ptitzler**
Adds H102: `Apache 2.0 license header not found` to ignore list for notebook files. It is ignored in other Elyra repos as well.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,6 +29,6 @@ lint-scripts: test-dependencies
 	flake8 .
 
 lint-notebooks: test-dependencies
-	nbqa flake8 */*.ipynb
+	nbqa flake8 --ignore=H102 */*.ipynb
```
Comment (lresende) [inline Makefile:32]: Can we centralize all configurations in .flake8 file

Comment (ptitzler) [reply Makefile:32]: We do want the license header in Python scripts!

**Comment by romeokienzler** [comment]
@ptitzler @lresende I've resolved the errors for claimed_utils.py in a branch already - maybe once I've fixed all linting errors yo can pull in the changes into ptitzler-patch-1?

**Comment by ptitzler** [comment]
>  I've resolved the errors for claimed_utils.py in a branch already

https://github.com/elyra-ai/component-library/pull/15/ still shows linting errors for the file therefore the PR was not merged

image

File: Makefile (hunk 1 above)
Comment (ptitzler) [reply Makefile:32]: I checked the documentation but couldn't find an alternative (e.g. in `.flake8`) that only ignores a specific warning/error for files with a specific extension.

**Pull request merged**